lib/engine/config/game/g_18_zoo.rb

```diff
@@ -0,0 +1,1231 @@
+# frozen_string_literal: true
+
+# rubocop:disable Lint/RedundantCopDisableDirective, Layout/LineLength, Layout/HeredocIndentation
+
+module Engine
```

Comment:
run this through a json formatter, there are too many white spaces

Reply:
before we launch, can we also include english translations?

Reply:
ok for the formatter, and sure, I have to put the English translation for all the text